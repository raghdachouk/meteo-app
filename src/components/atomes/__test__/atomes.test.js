import renderer from "react-test-renderer";

import Button from "../button";
import Input from "../input";

test("Button renders correctly", () => {
  const button = renderer.create(
    <Button
      disabled={false}
      type="submit"
      onClick={() => console.log("test snapshot clicked")}
    >
      Test
    </Button>
  );
  const tree = button.toJSON();
  expect(tree).toMatchSnapshot();
});

test("Input renders correctly", () => {
  const input = renderer.create(
    <Input name="test" placeholder="test snapshot"></Input>
  );
  const tree = input.toJSON();
  expect(tree).toMatchSnapshot();
});
