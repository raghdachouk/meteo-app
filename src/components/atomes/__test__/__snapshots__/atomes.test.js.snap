// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders correctly 1`] = `
<button
  className="button"
  disabled={false}
  onClick={[Function]}
>
  Test
</button>
`;

exports[`Input renders correctly 1`] = `
<div
  className="input"
>
  <input
    disabled={false}
    name="test"
    onChange={[Function]}
    placeholder="test snapshot"
    type="text"
  />
</div>
`;
